webapp/src/js/app.js

```diff
@@ -1,3 +1,10 @@
+// While we already do this earlier in inbox.js we have to check again for Karma
+// tests as they don't hit that code
+if (!window.startupTimes) {
+  window.startupTimes = {};
+}
+window.startupTimes.firstCodeExecution = performance.now();
```

Comment:
You might consider putting this in a <script> tag inside the `<head>` of `inbox.html` so that it includes time to download and parse all assets.